src/main/java/org/mockbukkit/mockbukkit/scheduler/BukkitSchedulerMock.java

```diff
@@ -165,12 +169,23 @@ public synchronized long getCurrentTick()
 		return currentTick;
 	}
 
-	/**
-	 * Perform one tick on the server.
-	 */
-	public synchronized void performOneTick()
+	private void processEntities()
 	{
-		currentTick++;
+		for (World world : Bukkit.getWorlds())
+		{
+			for (Entity entity : world.getEntities())
+			{
+				if (entity instanceof LivingEntityMock living)
```

Comment:
In the `EntityMock` class there's a tick method, maybe we could re-use that instead?